packages/@aws-cdk/aws-lambda/lib/scalable-version-attribute.ts

```diff
@@ -0,0 +1,65 @@
+import * as appscaling from '@aws-cdk/aws-applicationautoscaling';
+
+/**
+ * Interface for scalable attributes
+ */
+export interface IScalableVersionAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level
+   */
+  scaleOnUtilization(props: UtilizationScalingProps): void;
+  /**
+   * Add scheduled scaling for this scaling attribute
+   */
+  scaleOnSchedule(id: string, actions: appscaling.ScalingSchedule): void;
+}
+
+/**
+ * Properties for enabling Lambda utilization tracking
+ */
+export interface UtilizationScalingProps extends appscaling.BaseTargetTrackingProps {
+  /**
+   * Target utilization percentage for the attribute
+   */
+  readonly targetUtilizationPercent: number;
+}
+
+/**
+ * A scalable lambda version attribute
+ */
+export class ScalableVersionAttribute extends appscaling.BaseScalableAttribute {
+  /**
+   * Scale out or in to keep utilization at a given level
+   */
+  public scaleOnUtilization(props: UtilizationScalingProps) {
+    if (props.targetUtilizationPercent < 0 || props.targetUtilizationPercent > 1) {
+      throw new Error('target utilization is a percentage and must be between 0 and 1.');
+    }
+    super.doScaleToTrackMetric('Tracking', {
+      targetValue: props.targetUtilizationPercent,
+      predefinedMetric: appscaling.PredefinedMetric.LAMBDA_PROVISIONED_CONCURRENCY_UTILIZATION,
+      ...props,
+    });
+  }
+
+  /**
+   * Scale out or in based on time
+   */
+  public scaleOnSchedule(id: string, action: appscaling.ScalingSchedule) {
+    super.doScaleOnSchedule(id, action);
```

Comment:
The `id` becomes `scheduledActionName` which has a weird regex expression. I think it makes more sense to hardcode the `id` as `Scheduling`, since the other method has a hardcoded `id` of `Tracking`.